KsApi/Service.swift

```diff
@@ -106,6 +106,21 @@ public struct Service: ServiceType {
       .flatMap(CreatePaymentSourceEnvelope.producer(from:))
   }
 
+  public func triggerCapiEventInput(input: TriggerCapiEventInput)
+    -> SignalProducer<EmptyResponseEnvelope, ErrorEnvelope> {
+    return GraphQL.shared.client
+      .perform(mutation: GraphAPI
+        .TriggerCapiEventMutation(input: GraphAPI
+          .TriggerCapiEventInput(
+            projectId: input.projectId,
+            eventName: input.eventName,
+            externalId: input.externalId
+          )))
+      .flatMap { _ in
```

Comment:
No need create the closure with `_ in`, just using `producer(from:))` like above.

Reply:
Looking at the other examples that use `EmptyResponseEnvelope`, they do create this closure with a SignalProducer. I believe this is because `EmptyResponseEnvelope` doesn't already have a static `producer(from:)` function